go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/giantswarm/cluster-operator/v3
+module github.com/giantswarm/cluster-operator/v3/v3
```

Comment:
this seems wrong

Reply:
eagle eye! not sure how it ended up like that and why it does not complain locally